MKSOutputDevice.py

```diff
@@ -882,6 +882,121 @@ def resumePrint(self):
             self._ischanging = True
         self._sendCommand("M24")
 
+    def printer_set_connect(self):
+        self._sendCommand("M20")
+        self.setConnectionState(cast(ConnectionState, UnifiedConnectionState.Connected))
+        self.setConnectionText(self._translations.get("connected"))
+
+    def get_current_temp(self, temp):
+        return float(temp[0:temp.find("/")])
+
+    def get_target_temp(self, temp):
+        return float(temp[temp.find("/") + 1:len(temp)])
+
+    def printer_info_update(self, info):
+        printer = self.printers[0]
+        t0_temp = info[info.find("T0:") + len("T0:"):info.find("T1:")]
+        t1_temp = info[info.find("T1:") + len("T1:"):info.find("@:")]
+        bed_temp = info[info.find("B:") + len("B:"):info.find("T0:")]
+        printer.updateBedTemperature(self.get_current_temp(bed_temp))
+        printer.updateTargetBedTemperature(self.get_target_temp(bed_temp))
+        extruder0 = printer.extruders[0]
+        extruder0.updateHotendTemperature(self.get_current_temp(t0_temp))
+        extruder0.updateTargetHotendTemperature(self.get_target_temp(t0_temp))
+        if self._number_of_extruders > 1:
+            extruder1 = printer.extruders[1]
```

Comment:
What happens if we will have 3 or more extruders?
I think we can make a `forEach` loop, for all `printer.extruders` that we have.

Reply:
No, there can be only 2 extruders because if MKS WiFi protocol